### DISCUSSION TITLE
Plus and Minus inputs in Edit Portfolio Entry.

### DISCUSSION THREAD

**Discussion by WildBillDoE** [miguelmota/cointop#251] [Ideas]
Would it be possible to add + or - inputs in the Enter Holdings total so I don't have to break out a calculator every time to add or subtract coins from different exchanges?

Just an idea,
Thanks.

**Comment by lyricnz** [comment]
It actually already supports entering expressions. Try it!  See #110

**Reply by WildBillDoE** [reply]
Ahh yes, thank you.  Works great.

**Reply by lyricnz** [reply]
Thank you for your contribution. Happy to have people involved! Even better if you can code, or test, or document! :)

**Reply by WildBillDoE** [reply]
Can't code well or I would block buff doge coin ;) Somehow they have hijacked coingecko's Doge symbol or url.
JFYI: I'm running this on Rpi 4b running go.
go get has been depreciated for go install.
https://golang.org/doc/go-get-install-deprecation
